test/submit_coverage.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+# Create a symbolic link at src/main/scala/org to let coverage test retrieve all source,
+# and remove it after test
+# Ref: https://github.com/scoverage/sbt-coveralls
+ln -s src/main/scala/org ./
+COVERALLS_REPO_TOKEN=QknRiHqsMIzaOEbmSYtikFuxuVWEiPAJe sbt coveralls
```

Comment:
Why is the token exposed here?  To my understanding, we don't need it when running through TravisCI.
Please remove the line, and reset the token since it's been compromised.